demo/full/scripts/controllers/ContentList.tsx

```diff
@@ -753,9 +795,9 @@ function ContentList({
           />
         </div>
       </div>
-      {isCustomContent || (isLocalContent && isSavingOrUpdating) ? (
+      {(isCustomContent || (isLocalContent && isSavingOrUpdating)) && (
         <div className="custom-input-wrapper">
-          {isSavingOrUpdating ? (
+          {isSavingOrUpdating && (
```

Comment:
I find this clever JS trick (`boolean && actual value we want`) to be less readable and more risky than  a ternary `boolean ? actual value we want : null`.

Not you? Also applies to many other values here (`isLocalContent`, `shouldDisplayDRMSettings` etc.)

Reply:
I'm fine with both, I think react doc mention both, it's just that the `&&` is less verbose 😄.
Will change that